docs/installation-guide/quickstart.rst

```diff
@@ -0,0 +1,345 @@
+    kube-system   metalk8s-ui-656f6857b-cdt5p                   1/1       Running   0          18m       10.233.0.3     bootstrap.novalocal   <none>
+    kube-system   package-repositories-bootstrap                1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   registry-bootstrap                            1/1       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+    kube-system   salt-master-bootstrap                         2/2       Running   0          17m       172.21.254.7   bootstrap.novalocal   <none>
+
+
+Adding a master to the cluster
+------------------------------
+
+Now it's time to add more node to the cluster. First you need to secure the
+cluster by adding 2 nodes with etcd and master roles.
+Here is the procedure to add one, simply do it twice to have 3
+masters (bootstrap + 2 new master).
+
+1. Copy the ssh-key to the new node
+
+   .. code-block:: shell
+
+    $ ssh-copy-id <IP-of-the-new-master-node>
+
+2. Create a yaml config file for this new master.
+
+   .. code-block:: yaml
+
+    apiVersion: v1
+    kind: Node
+    metadata:
+      name: <new-master-node-name>
+      annotations:
+        metalk8s.scality.com/ssh-key-path: /etc/metalk8s/pki/id_rsa
+        metalk8s.scality.com/ssh-host: <IP-of-the-new-master-node>
+        metalk8s.scality.com/ssh-sudo: 'false'
+      labels:
+        metalk8s.scality.com/version: '2.0'
+        node-role.kubernetes.io/master: ''
+        node-role.kubernetes.io/etcd: ''
+    spec:
+      taints:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/etcd
+
+3. Declare the new master node into the cluster.
+
+   .. code-block:: shell
+
+    $ kubectl --kubeconfig /etc/kubernetes/admin.conf apply -f new-master-node.yaml
+    node/new-master-node created
+
+4. Check that the new master node is correctly inserted into the cluster.
+
+   .. code-block:: shell
+
+    $ kubectl get node
+    NAME                   STATUS    ROLES                         AGE       VERSION
+    bootstrap              Ready     bootstrap,etcd,infra,master   12d       v1.11.7
+    new-master-node        Unknown   etcd,master                   29s
+
+5. The new master node now need to be installed to change its status from Unknown to Ready.
+   You go into the master-bootstrap pod ...
+
+   .. code-block:: shell
+
+    $ kubectl -ti -n kube-system exec salt-master-bootstrap bash
+
+   Try first to ping the new master node ...
+
+   .. code-block:: shell
+
+    $ salt-ssh -i --roster kubernetes <new-master-node-name> test.ping
+    <new-master-node-name>:
+        True
+
+   Launch the command to perform the installation
+
+   .. code-block:: shell
+
+    $ salt-run state.orchestrate metalk8s.orchestrate.deploy_new_node saltenv=metalk8s-2.0 \
+      pillar="{'bootstrap_id': 'bootstrap', 'node_name': '<new-master-node-name>'}"
```

Comment:
Bootstrap_id no longer needed and `node_name` should be in `orchestrate` section
```suggestion
      pillar="{'orchestrate:' {'node_name': '<new-master-node-name>'}}"
```